UI/src/view/point_asset/pointAssetStyle.js

```diff
@@ -15,15 +15,15 @@
                 icon: {
                     scale : 0.75,
                     rotateWithView: true,
-                    rotation: '${rotation}',
+                    rotation: '',
```

Comment:
Take out rotation, I think there is no use in this

Reply:
Applied